<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardReader::services::RacomClientWidget</class>
 <widget class="QWidget" name="CardReader::services::RacomClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Racom Client</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QSpinBox" name="edSirxdDataListenPort">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1025</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="chkListenSirxdData">
     <property name="text">
      <string>sirxd data</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Listen on UDP port</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="chkListenRawData">
     <property name="text">
      <string>Raw SI data</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="edRawDataListenPort">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1024</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Listen on TCP port</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkListenRawData</tabstop>
  <tabstop>edRawDataListenPort</tabstop>
  <tabstop>chkListenSirxdData</tabstop>
  <tabstop>edSirxdDataListenPort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
